pyproject.toml

```diff
@@ -1,2 +1,3 @@
 [build-system]
-requires = ["setuptools>=30.3", "wheel", "setuptools_scm>=1.15"]
+requires = ["setuptools>=34.4", "wheel", "setuptools_scm>=1.15"]
+build-backend = 'setuptools.build_meta'
```

Comment:
Different quotes style

Reply:
Yeah, maybe that should be consistent. I'm pretty sure I copied these two different concepts from two separate sources. I can make them consistent in skeleton. [This reference](https://snarky.ca/clarifying-pep-518/) suggests double-quotes for everything.

Reply:
Yeah, I think I only saw people using double quotes in toml files so far.